src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -150,4 +150,13 @@ public static int parseReviewRating(String reviewRating) throws ParseException {
         }
         return Integer.valueOf(trimmedRating);
     }
+
+    /**
+     * Parses {@code reviewDate} to String value corresponding to date.
+     */
+    public static String parseReviewDate(String reviewDate) {
+        requireNonNull(reviewDate);
+        String trimmedDate = reviewDate.trim();
+        return trimmedDate;
+    }
```

Comment:
oh? wouldn't it be better if we store the date as DateTime objects instead?

Reply:
I feel string is better as it aligns with the other code. Like phone number is stored as string.

Reply:
phone number makes sense to be stored as string though? i feel that we might need to manipulate the date, so it'll make more sense if you stored it as a DateTime object? i think you can look into using the SimpleDateFormat we used for the ip